Sources_v3/Database/DTOs/UserDTO_Tests.swift

```diff
@@ -82,6 +82,33 @@ class UserDTO_Tests: XCTestCase {
         }
     }
     
+    func test_defaultExtraDataIsUsed_whenExtraDataDecodingFails() throws {
+        let userId: UserId = .unique
+        
+        let payload: UserPayload<DefaultExtraData.User> = .init(
+            id: userId,
+            role: .admin,
+            createdAt: .unique,
+            updatedAt: .unique,
+            lastActiveAt: .unique,
+            isOnline: true,
+            isInvisible: true,
+            isBanned: true,
+            teams: [],
+            extraData: .defaultValue
+        )
+        
+        try database.writeSynchronously { session in
+            // Save the user
+            let userDTO = try! session.saveUser(payload: payload)
+            // Make the extra data JSON invalid
+            userDTO.extraData = #"{"invalid": json}"# .data(using: .utf8)!
+        }
+        
+        let loadedMessage: ChatUser? = database.viewContext.user(id: userId)?.asModel()
```

Comment:
```suggestion
        let loadedUser: ChatUser? = database.viewContext.user(id: userId)?.asModel()
```

Reply:
Fixed in 0ae98e4ceba2bd6e34439ba1f8b42d91c1b1fde4